documentation/htc_workloads/specific_resources/gpu-jobs.md

```diff
@@ -25,73 +28,93 @@ attributes). For example:
+request these capabilities, `require_gpus`. A few attributes that may
+be useful: 
 
-Any of the attributes above can be used in the submit file's `requirements` line to 
-select a specific kind of GPU. For 
-example, to request a GPU with more than 8GB of GPU memory, one could use: 
+  * `Capability`: this is NOT the GPU library, but rather a measure of the GPU's "Compute Capability"
+  * `DriverVersion`: maximum version of the GPU libraries that can be supported
+  * `GlobalMemoryMb`: amount of GPU memory available on the GPU device
 
-    requirements = (GPUs_GlobalMemoryMb >= 8192)
-    
 If you want a certain type or family of GPUs, we usually recommend using the GPU's 
-'Compute Capability', known as the `GPUs_Capability` by HTCondor. An A100 GPU has a 
+'Compute Capability', known as the `Capability` by HTCondor. An A100 GPU has a 
 Compute Capability of 8.0, so if you wanted to run on an A100 GPU specifically, 
 the submit file requirement would be: 
 
-    requirements = (GPUs_Capability == 8.0)
+    require_gpus = (Capability >= 8.0)
+
+Multiple requirements can be specified by using && statements:
+
+    require_gpus = (Capability >= 7.5) && (GlobalMemoryMb >= 11000)
 
 **Note that the more requirements you include, the fewer resources will be available 
 to you! It's always better to set the minimal possible requirements (ideally, none!) 
 in order to access the greatest amount of computing capacity.**
 
-# Available GPUs
+### Sample Submit File
+
+    universe = container
+    container_image = /cvmfs/singularity.opensciencegrid.org/htc/tensorflow:1.3
+
+    log = job_$(Cluster)_$(Process).log
+    error = job_$(Cluster)_$(Process).err
+    output = job_$(Cluster)_$(Process).out
+    
+    executable = run_gpu_job.py
+    #arguments = 
+   
+    # specify both general requirements and gpu requirements 
+    requirements = True
+    require_gpus = (Capability > 7.5)
+    
+    request_gpus = 1
+    request_cpus = 1
+    request_memory = 4GB
+    request_disk = 4GB
+    
+    queue 1
 
-## Capacity
+## Available GPUs
 
-There are multiple OSPool contributors providing GPUs on a regular basis to the OSPool. 
-Some of these contributors will make their GPUs available only when there is demand 
-in the job queue, so after initial small-scale job testing, we strongly recommend 
-submitting a signficant batch of test jobs to explore 
-how much throughput you can get in the system as a whole. 
+### Capacity
 
-Also, **some members of the OSPool will only run GPU jobs if the job 
-uses a container** (see note about software below). 
+There are multiple OSPool contributors providing GPUs on a regular
+basis to the OSPool. Some of these contributors will make their GPUs
+available only when there is demand in the job queue, so after initial
+small-scale job testing, we strongly recommend submitting a signficant
+batch of test jobs to explore how much throughput you can get in the
+system as a whole.
```

Comment:
```suggestion
system as a whole. As a reminder, because the OSPool is dynamic, the more jobs submitted requesting GPUs, the more GPU machines will be pulled into the OSPool as execution points.
```